### PR TITLE
feat: add role for token

### PR DESCRIPTION
**Pull request by pggb25**

### CODE REVIEW

**Review by erebe — APPROVED** [review]

File: src/schemas/OrganizationApiTokenCreateRequest.yaml [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
   name:
     type: string
   description:
     type: string
   scope:
     $ref: './enums/OrganizationApiTokenScope.yaml'
+  roleId:
+    type: string
+    format: uuid
+    nullable: true
```
Comment (acarranoqovery) [inline src/schemas/OrganizationApiTokenCreateRequest.yaml:15]: description: the roleId provided by the "List organization custom roles" endpoint.

File: src/schemas/enums/OrganizationApiTokenScope.yaml
Comment (acarranoqovery) [inline src/schemas/enums/OrganizationApiTokenScope.yaml:1]: can we just delete this since we removed the enums from the OrganizationApiTokenResponse.yaml?

**Review by acarranoqovery — CHANGES_REQUESTED** [review]

File: src/schemas/OrganizationApiTokenCreateRequest.yaml (hunk 1 above)
Comment (pggb25) [reply src/schemas/OrganizationApiTokenCreateRequest.yaml:15]: done

File: src/schemas/enums/OrganizationApiTokenScope.yaml
Comment (pggb25) [reply src/schemas/enums/OrganizationApiTokenScope.yaml:1]: still use when create token as deprecated. Can be removed once front will use the roleId

**Review by acarranoqovery — APPROVED** [review]

**Pull request merged**